build/liqo-controller-manager/Dockerfile

```diff
@@ -1,6 +1,13 @@
 FROM alpine:3.9.5
 RUN apk add --update curl bash openssl
-RUN curl -LO https://storage.googleapis.com/kubernetes-release/release/v1.17.0/bin/linux/amd64/kubectl && chmod +x ./kubectl && cp kubectl /usr/bin/kubectl
+RUN <<EOR
+  case "`uname -i`" in
+  x86_64|amd64)
+    curl -LO https://storage.googleapis.com/kubernetes-release/release/v1.17.0/bin/linux/amd64/kubectl && chmod +x ./kubectl && cp kubectl /usr/bin/kubectl
```

Comment:
Could it be worth to upgrade the kubectl version? In any case, a fixed version could lead to problems in the future.
Additionally, it could be better to just set a variable and then run the command, to avoid code duplication which can easily become inconsistent.